src/nycdb/database.py

```diff
@@ -30,6 +30,30 @@ def insert(self, row):
             curs.execute(SQL, row)
         self.conn.commit()
 
+    def insert_rows_quickly(self, rows, table_name=None):
+        """
+        Inserts many row, all in the same transaction, as one big
+        SQL statement.
+        """
+
+        if table_name is None:
+            table_name = self.table_name
+
+        with self.conn.cursor() as curs:
+            big_sql = sql.insert_many(table_name, rows, curs)
+            try:
+                curs.execute(big_sql)
+            except psycopg2.DataError:
+                print("An error occurred. Inserting the rows one-by-one, ")
+                print("which should provide more helpful information.")
+                # This should throw an exception, but with more helpful
+                # logging.
+                self.insert_rows(rows, table_name)
```

Comment:
I figure we could keep the old `insert_rows` around to make debugging easier in case one of the rows trips up postgres, since we could just print out the one row that caused things to fail instead of showing 1000.  However, if we leave this in, I should probably add a test for it.